Unit-02/01-blueprints/project/static/script.js

```diff
@@ -0,0 +1,15 @@
+$(document).ready(function(){
+    function length_less_than_six(string){
+    	if(string.length<=6) return true
+    	return false
+    }
+
+    $("input").on('keyup', function(e){
```

Comment:
You can't just target an `input` here since it will just find things like your hidden input and console.log the CSRF Token